stl/inc/ranges

```diff
@@ -2327,6 +2382,11 @@ namespace ranges {
     template <class _Rng, class _Pr>
     filter_view(_Rng&&, _Pr) -> filter_view<views::all_t<_Rng>, _Pr>;
 
+#if _HAS_CXX23
+    template <class _Rng, class _Pr>
+    inline constexpr auto _Compile_time_max_size<filter_view<_Rng, _Pr>> = _Compile_time_max_size<_Rng>;

```

Comment:
Why does `filter_view` not have the `const` specialization as well? (same question below with, for example, `drop_while_view`)

Reply:
Those views are not const-iterable.